docs/transpile/transpile-with-pass-managers.ipynb

```diff
@@ -0,0 +1,483 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Transpile with pass managers\n",
+    "\n",
+    "The recommended way to transpile a circuit is to create a staged pass manager and then execute its `run` method with your circuit as input. This page explains how to transpile your quantum circuits this way."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## What is a (staged) pass manager?\n",
+    "\n",
+    "In the context of Qiskit, transpilation refers to the process of transforming an input circuit into a form that is suitable to be executed on a quantum device. Transpilation typically occurs in a sequence of steps called transpiler passes. The circuit is processed by each transpiler pass in sequence, with the output of one pass becoming the input to the next. For example, one pass could go through the circuit and merge all consecutive sequences of single-qubit gates, and then the next pass could synthesize these gates into the basis set of the target device. The transpiler passes included with Qiskit are located in the [qiskit.transpiler.passes](/api/qiskit/transpiler_passes) module.\n",
+    "\n",
+    "A pass manager is an object that stores a list of transpiler passes and can execute them on a circuit. You create a pass manager by initializing a [PassManager](/api/qiskit/qiskit.transpiler.PassManager) with your list of transpiler passes. To run the transpilation on a circuit, call the [run](/api/qiskit/qiskit.transpiler.PassManager#run) method with your circuit as input.\n",
+    "\n",
+    "A staged pass manager is a special kind of pass manager that represents a level of abstraction above that of a normal pass manager. While a normal pass manager is composed of a number of transpiler passes, a staged pass manager is composed of a number of *pass managers*. This is a useful abstraction because transpilation is typically happens in discrete stages, as described in [Transpiler stages](transpiler-stages), with each stage being represented by a pass manager. The rest of this page will describe how to create and customize (staged) pass managers."
```

Comment:
```suggestion
    "A staged pass manager is a special kind of pass manager that represents a level of abstraction above that of a normal pass manager. While a normal pass manager is composed of several transpiler passes, a staged pass manager is composed of several *pass managers*. This is a useful abstraction because transpilation typically happens in discrete stages, as described in [Transpiler stages](transpiler-stages), with each stage being represented by a pass manager. The rest of this page describes how to create and customize (staged) pass managers."
```